Sources/SwiftCrossUI/Views/Modifiers/PresentationModifiers.swift

```diff
@@ -0,0 +1,67 @@
+extension View {
+    /// Sets the available detents (heights) for a sheet presentation.
+    ///
+    /// This modifier only affects the sheet presentation itself when applied to the
+    /// top-level view within a sheet. It allows users to resize the sheet to different
+    /// predefined heights.
```

Comment:
Remove the part about top-level views (not true anymore with my suggested change to the preference value merging implementation).